incompatibility/main.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
Is this file still wanted since you are going to move forward with a single plugin approach?

Reply:
removed